src/test/java/com/fasterxml/jackson/databind/deser/builder/BuilderSimpleTest.java

```diff
@@ -430,4 +479,66 @@ public void testPrivateInnerBuilder() throws Exception
         Value2354 result = MAPPER.readValue(json, Value2354.class);
         assertEquals(13, result.value());
     }
+
+    public void testSuccessfulValidatingBuilder() throws Exception
+    {
+        ValidatingValue result = MAPPER.readValue(aposToQuotes("{'a':'1','b':'2'}"), ValidatingValue.class);
+        assertEquals("1", result.first);
+        assertEquals("2", result.second);
+    }
+
+
+    public void testFailingValidatingBuilderWithExceptionWrapping() throws Exception
+    {
+        ObjectMapper withWrapping = MAPPER.isEnabled(DeserializationFeature.WRAP_EXCEPTIONS)
+                ? MAPPER : MAPPER.copy().enable(DeserializationFeature.WRAP_EXCEPTIONS);
+        try {
+            withWrapping
+                    .readValue(aposToQuotes("{'a':'1'}"), ValidatingValue.class);
+            fail("Expected an exception");
+        } catch (JsonMappingException e) {
```

Comment:
Should this verify that we get `ValueInstantiationException` (given that we know nature of specific fail)?

Reply:
N/m, I can change that.

Reply:
You’re right, thanks!